kafka_scanner/tests/__init__.py

```diff
@@ -16,16 +17,25 @@
 ConsumerRecord = namedtuple("ConsumerRecord", ["partition", "offset", "key", "value"])
 
 
-def get_kafka_msg_samples(msgs=None, fetch_count=0):
+def get_kafka_msg_samples(msgs=None, fetch_count=0, msgformat='msgpack', compress=True):
     if not msgs:
-        msgs = [('AD12345', "my-body"),
-                ('AD34567', "second my-body"),
```

Comment:
Can you separate pylint style fixes and new feature in different commits?

Reply:
oh ok, do you guys dont mind if there are multiple commits per PR? or should i remove it completely and then do another PR?

Reply:
I don't mind multiple commits per PR. Not needed an extra PR.